config/manager/kustomization.yaml

```diff
@@ -1,6 +1,5 @@
 resources:
 - manager.yaml
-- clustermanagementaddon.yaml
```

Comment:
Looking in the rest of this file, I see that this line was making a difference.

Reply:
@MikeSpreitzer I'm not sure what you are saying.... that I should put it back.
May be I removed before I figure out the pkg/internal/kubebuilder-gen.go.
I put it back.

Reply:
@MikeSpreitzer I now realie that this was removed because the clustermanagementaddon.yaml
 CR is now implemented by the init container

Reply:
Yes, but if the kustomization is no longer applied to the ClusterManagementAddOn object YAML then this is making a functional difference.